factfinder/main.py

```diff
@@ -296,6 +303,31 @@ def calculate_e_m(self, pff_variable: str, geotype: str) -> pd.DataFrame:
         df = aggregate_vertical(df)
         return df
 
+    def calculate_e_m_p_z(self, pff_variable:str, geotype:str) -> pd.DataFrame:
+        """
+        This function is used for calculating profile variables only with 
+        non-aggregated-geography geotype
+        """
+         # 1. create variable
+        v = self.create_variable(pff_variable)
+        # hard coding because by definition profile-only
+        #  variables only has 1 census variable
+        census_variable = v.census_variable[0]
```

Comment:
It isn't possible that we'd have sums of profile variables in the future? This function is for variables where there is neither horizontal nor vertical aggregation?

Reply:
It looks like lines 483 and 484 are written to allow for download of multiple profile variables.

Reply:
If we have sum of profile variables, then they should go into the normal processing right, because in that case we would need to actually calculate the p and z. the profile-only-variables is created just so we can take PE PM directly from census

Reply:
but you are right, we can be more direct here 
```python
census_variable = v.census_variable[0]
E_variables = census_variable+'E'
M_variables =  census_variable+'M'
PE_variables =  census_variable+'PE'
PM_variables =  census_variable+'PM'
```